CareKitFHIR/CareKitFHIR.xcodeproj/project.pbxproj

```diff
@@ -430,7 +430,9 @@
 				MTL_ENABLE_DEBUG_INFO = INCLUDE_SOURCE;
 				MTL_FAST_MATH = YES;
 				ONLY_ACTIVE_ARCH = YES;
-				SDKROOT = iphoneos;
+				SDKROOT = auto;
```

Comment:
These types of changes are similar across all `xcodeproj` files and allow the projects and respective targets to build for iPhone and watchOS